mantidimaging/gui/windows/main/view.py

```diff
@@ -78,6 +79,8 @@ def __init__(self):
             self.setWindowTitle("Mantid Imaging Unstable")
             self.setWindowIcon(QIcon("./images/mantid_imaging_unstable_64px.png"))
 
+        self.use_cuda = cuda_is_present()
```

Comment:
I don't like this being here, but I also can't think of a better place as it's used in multiple windows. The model is possible, but then we'd just need a bunch of getters to reach the same thing..

Any ideas are welcome, if not I'm happy to approve with the current state

Reply:
Might be cleaner if `ReconstructWindowModel` (and anywhere else that want to know) calls `cuda_is_present()` itself. `cuda_check.py` could store the answer, so it does not need to repeat the check.